README.md

```diff
@@ -588,7 +588,7 @@ Other Style Guides
     }
 
     // good
-    const foo = function bar() {
+    const foo = function foo () {
```

Comment:
Primarily, it makes it easier to locate when you `grep` for the name of the function as read in a stack trace - it also lets you use a short, simple name for the local variable, but a long, verbose name for the function, for better debugging.